policyengine_us/parameters/gov/states/al/tax/income/exemptions/dependent.yaml

```
@@ -0,0 +1,27 @@
+description: Alabama provides dependent exemption amounts based on the filed AGI amount.
+
+brackets:
+- threshold:
+      2022-01-01: -.inf
+  amount: 
+      2022-01-01: 1_000
+- threshold:
+      2022-01-01: 50_001
```

You can use round numbers from the law here and then `right=True` in `.calc` in the Python (see [docs](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#creating-scales))
```suggestion
      2022-01-01: 50_000
```